src/actions/search.js

```diff
@@ -78,7 +83,19 @@ export function searchPostsWithParams(teamId, params, includeDeletedChannels = f
 }
 
 export function searchPosts(teamId, terms, isOrSearch, includeDeletedChannels) {
-    return searchPostsWithParams(teamId, {terms, is_or_search: isOrSearch}, includeDeletedChannels);
+    return searchPostsWithParams(teamId, {terms, is_or_search: isOrSearch, include_deleted_channels: includeDeletedChannels, page: 0, per_page: 20});
+}
+
+export function getMorePostsForSearch() {
+    return async (dispatch, getState) => {
+        const teamId = getCurrentTeamId(getState());
+        const {params, isEnd} = getState().entities.search.current[teamId];
+        if (!isEnd) {
+            const newParams = Object.assign({}, params);
+            newParams.page = newParams.page + 1;
+            await searchPostsWithParams(teamId, newParams)(dispatch, getState);
```

Comment:
Should this be returning the results of searchPostsWithParams?

Reply:
Ya I guess it should.